### DISCUSSION TITLE
Make 3.0 parts of the site (maybe including editor) to look like 2.0

### DISCUSSION THREAD

**Discussion by SJRNoodles** [ScratchAddons/ScratchAddons#1547] [Ideas]
For nostaliga

MOD NOTE: edited the title so that it makes sense (previously said "to look like 3.0")

**Comment by cobaltt7** [comment]
You mean to look like 2.0? That's a dupe
________________________________

**Reply by mxmou** [reply]
Yes, this was already suggested in #985.
